app/src/main/java/com/infomaniak/mail/utils/WebViewUtils.kt

```diff
@@ -65,9 +76,15 @@ class WebViewUtils(context: Context) {
         registerScript(jsBridgeScript)
         registerBodyEncapsulation()
         registerBreakLongWords()
+        if (isForPrint) registerIsForPrint()
```

Comment:
This is not common between the message html and the signature html, this should not be grouped inside this method